### ISSUE TITLE
[Technical Discovery] Chronological order of appeals / claims

### ISSUE THREAD

**Issue by kevinstachura** [department-of-veterans-affairs/va.gov-team#3119]
## User Story
As a claimant, I want to have a visual representation of the order in which I submitted my claims so that can see them in an organized fashion.

## Goal
List the appeals / issues within the Claim Status Tool in chronological order (order by date).

## Tasks
- [ ] Identify how the current organization occurs (e.g., from date submitted? from date when addition evidence is requested? etc.)
- [ ] Identify if user selecting the way claims are sorted is an option (design ticket needed)

## Acceptance Criteria
- [ ] Understand technical implications of listing in chronological order

**Comment by ndsprinkle** [comment]
@YanaRoy @kevinstachura 

I have investigated the appeals/claims and have some interesting finds. Currently, the list of appeals/claims on /track-claims/your-claims IS actually being sorted. They are being sorted by which appeal/claim was last updated. This sounds like a useful way to sort them, but seems quite confusing to a user considering the only date we display is the date the appeal/claim was received, which makes the list appear to have no order at all.

My initial hunch was also true and this is simply some front end side sorting going on, so changing the sort or adding more sorting options should be straightforward. While I think that offering the user the choice to change how this list is sorted would be useful, I think the higher priority would simply be changing the sorting method to something the user is visually aware of. This could either be accomplished by actually putting the updated date on the appeal/claim boxes in the list or even by just indicating "This list is being sorted by most recently updated" so the user is aware there is an order.

One other thing I wanted to note was that there is another place within the app that lists the veteran's appeals/claims. This lives under /my-va and only appears after the veteran selects "Disability compensation" and/or "Appeals" on /my-va/find-benefits and saves preferences. This list of appeals/claims is limited to those that have been updated in the last 30 days. I believe this list is sorted by the date the appeal/claim was received, but there isn't any front end side sorting going on, so I'm just assuming it is sorted by that externally.
